runtime/oti/vmconstantpool.xml

```diff
@@ -154,6 +154,8 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 	<classref name="openj9/internal/criu/JVMRestoreException" flags="opt_criuSupport"/>
 
 	<!-- Class references needed to support Foreign Linker API. -->
+	<classref name="java/lang/invoke/NativeMethodHandle" flags="opt_openjdkFfi" versions="22-"/>
```

Comment:
What is the point of elements with `flags="opt_openjdkFfi"` because `computeJCLRuntimeFlags()` never sets `JCL_RTFLAG_OPT_OPENJDK_FFI`?

Reply:
Yes as I explained previously that this is our intention to set the flag to enable all these resolution of these entries are skipped at startup because the the FFI classes in JDK17 is still in the `incubator` rather than `java.base`  (given `vmconstantpool` is only used to resolve the entries in `java.base`) . We will resolve them later in our own code later when the FFI API are invoked in applications, in which case `JCL_RTFLAG_OPT_OPENJDK_FFI` should not be set.